### PR TITLE
Document handler and mutator assets

### PR DESCRIPTION
**Pull request by nikkictl**
Signed-off-by: Nikki Attea <nikki@sensu.io>

## Description
Documentation for new handler/mutator assets feature.
Requires https://github.com/sensu/sensu-go/pull/2182.

## Motivation and Context
Closes https://github.com/sensu/sensu-docs/issues/814

### CODE REVIEW

**Review by palourde — APPROVED** [review]

File: content/sensu-core/2.0/reference/assets.md [hunk 1]
```diff
@@ -14,16 +14,32 @@ menu:
...
+
+### Default cache directory
+
+system  | sensu-backend                               | sensu-agent
+--------|---------------------------------------------|-------------
+default | `/var/cache/sensu/sensu-backend`            | `/var/cache/sensu/sensu-agent`
+Windows | `C:\\ProgramData\sensu\cache\sensu-backend` | `C:\\ProgramData\sensu\cache\sensu-agent`
+
+If the requested asset is not in the local cache, it is downloaded from the asset
+url. The Sensu backend does not currently provide any storage for assets; they
```
Comment (apaskulin) [inline content/sensu-core/2.0/reference/assets.md]:
```suggestion
URL. The Sensu backend does not currently provide any storage for assets; they
```

File: content/sensu-core/2.0/guides/install-check-executables-with-assets.md [hunk 2]
```diff
@@ -79,9 +79,76 @@ $ sensuctl event list
...
+
+{{< highlight shell >}}
+$ sensuctl handler create influx-db \
+  --command "sensu-influxdb-handler --addr 'http://123.4.5.6:8086' --username 'foo' --password 'bar' --db-name 'myDB'" \
+  --runtime-assets sensu-influxdb-handler
+{{< /highlight >}}
+
+### Validating the handler asset
+
+Once the handler is setup, it must be attached to a check to handle the event output (ex. `collect-metrics.sh`).
```
Comment (apaskulin) [inline content/sensu-core/2.0/guides/install-check-executables-with-assets.md]: Is `collect-metrics.sh` the name of the check here? Would `collect-metrics` be better?

**Review by apaskulin — APPROVED** [review]
Awesome work! So excited for this feature 🎉

File: content/sensu-core/2.0/guides/install-check-executables-with-assets.md (hunk 2 above)
Comment (nikkictl) [reply content/sensu-core/2.0/guides/install-check-executables-with-assets.md]: Yes, that's right.. and yes that would be better!

**Review by apaskulin — APPROVED** [review]

**Pull request merged**